client/lib/redux-bridge/index.js

```diff
@@ -0,0 +1,15 @@
+let reduxStore = null;
+
+export function setReduxStore( store ) {
+	reduxStore = store;
+}
+
+/**
+ * Dispatch an action against the current redux store
+ */
+export function reduxDispatch( ...args ) {
```

Comment:
yeah, was thinking about that... another pr though. exposing state feels even more dirty somehow. probably because that implies that the consumer should also be a listener to the store maybe... dunno.